Resources/public/js/views/universaldiscovery/ez-universaldiscoverysearchview.js

```diff
@@ -0,0 +1,500 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-universaldiscoverysearchview', function (Y) {
+    "use strict";
+    /**
+     * Provides the universal discovery search method
+     *
+     * @module ez-universaldiscoverysearchview
+     */
+    Y.namespace('eZ');
+
+    var events = {
+            '.ez-ud-search-button': {
+                'tap': '_search',
```

Comment:
I have added `keydown` event on on the search text input. I'm wondering if there is possibility to triggerr the event only if enter was pushed without checking it inside event handler?